scripts/partition_manager/partition_manager.rst

```diff
@@ -328,6 +331,100 @@ It includes :file:`autoconf.h` (which is generated by Kconfig) and uses a Kconfi
 
    #endif /* CONFIG_SOC_NRF9160 */
 
+.. _pm_regions:
+
+Regions
+=======
+
+The Partition Manager places partitions in different *regions*.
+For example, you can use regions for internal flash memory and external flash memory.
+These regions are predefined as ``flash_primary`` and ``external_flash``, respectively.
+
+To define in which region a partition should be placed, use the ``region`` property in the configuration of the partition.
+If no region is specified, the internal flash region is used.
+
+Defining a region
+-----------------
+
+Each region is defined by a name, a start address, a size, a placement strategy, and, if applicable, a device name.
+A region only specifies a device name if there is a device driver associated with the region, for example, a driver for an external SPI flash.
+
+There are three types of placement strategies, which affect how partitions are placed in regions:
+
+start_to_end
+   Place partitions sequentially from start to end.
+   Partitions stored in a region with this placement strategy cannot affect their placement through the ``placement`` property.
+   The unused part of the region is assigned to a partition with the same name as the region.
+
+end_to_start
+   Place partitions sequentially from end to start.
+   Partitions stored in a region with this placement strategy cannot affect their placement through the ``placement`` property.
+   The unused part of the region is exposed through a partition with the same name as the region.
+
+complex
+   Place partitions according to their ``placement`` configuration.
+   By default, the unused part of the region is exposed through a partition named ``app``.
+   This name can be changed through a command line argument.
```

Comment:
How? Can we point somewhere for more information?

Reply:
Hm, this feature is actually not used now, and I see no reason why we should document it. I will change it to say that the unused part is called `app`.